test_fms/parser/generic_blocks.F90

```diff
@@ -0,0 +1,111 @@
+program generic_blocks
```

Comment:
This is just missing the GNU license header

Reply:
Fix pulled in from @uramirez8707's `generic_parser` branch.